feelslikemonday/app/src/androidTest/java/com/example/feelslikemonday/DAO/FollowPermissionDAOTest.java

```diff
@@ -49,6 +59,10 @@ public void onCallback() {
                 signal.countDown();
             }
         });
+
+
```

Comment:
We don't need to necessarily commit all of these whitespace changes. AndroidStudio can show you what changes occurred in what files when committing; this can help to keep track of what has been changed